src/rbr/RoutingTableGenerator.java

```diff
@@ -87,33 +87,32 @@ private int maxOfRegions() {
     }
 
     private void PrintRegions(Vertex router, BufferedWriter bw) throws IOException {
-        bw.append("\n -- Router " + router.name() + "\n");
-        bw.append("(");
-
-        for (int regionIndex = 0; regionIndex < regionsForVertex.get(router).size(); regionIndex++) {
-            Region currentRegion = regionsForVertex.get(router).get(regionIndex);
-            int Xmin = currentRegion.box().min(0);
-            int Ymin = currentRegion.box().min(1);
-            int Xmax = currentRegion.box().max(0);
-            int Ymax = currentRegion.box().max(1);
-
-            String region = "(\""
-                    + opToBinary(regionsForVertex.get(router).get(regionIndex).inputPorts())
-                    + intToBinary(Xmin, bitPerCoordinate)
-                    + intToBinary(Ymin, bitPerCoordinate)
-                    + intToBinary(Xmax, bitPerCoordinate)
-                    + intToBinary(Ymax, bitPerCoordinate)
-                    + opToBinary(regionsForVertex.get(router).get(regionIndex).outputPorts())
-                    + "\")";
-
-            bw.append(region);
-            if(regionIndex < maxOfRegions - 1)
-                bw.append(",");
-            bw.newLine();
-        }
-        paddingToMaxRegions(bw, router);
-
-        bw.append(")");
+         bw.append("\n -- Router " + router.name() + "\n");
+         bw.append("(");
+         for (int regionIndex = 0; regionIndex < regionsForVertex.get(router).size(); regionIndex++) {
+             Region currentRegion = regionsForVertex.get(router).get(regionIndex);
+             int Xmin = currentRegion.box().min(0);
+             int Ymin = currentRegion.box().min(1);
+             int Xmax = currentRegion.box().max(0);
+             int Ymax = currentRegion.box().max(1);
+
+             String region = "(\""
+                     + opToBinary(regionsForVertex.get(router).get(regionIndex).inputPorts())
+                     + intToBinary(Xmin, bitPerCoordinate)
+                     + intToBinary(Ymin, bitPerCoordinate)
+                     + intToBinary(Xmax, bitPerCoordinate)
+                     + intToBinary(Ymax, bitPerCoordinate)
+                     + opToBinary(regionsForVertex.get(router).get(regionIndex).outputPorts())
+                     + "\")";
+
+             bw.append(region);
+             if(regionIndex < maxOfRegions - 1)
+                 bw.append(",");
+             bw.newLine();
+         }
+         paddingToMaxRegions(bw, router);
+
+         bw.append(")");
```

Comment:
Seems that above lines (90 to 115) has same problem as above comment. Please, remove the extra space.